docs/ext/commands/commands.rst

```diff
@@ -585,6 +585,60 @@ handlers that allow us to do just that. First we decorate an error handler funct
 The first parameter of the error handler is the :class:`.Context` while the second one is an exception that is derived from
 :exc:`~ext.commands.CommandError`. A list of errors is found in the :ref:`ext_commands_api_errors` page of the documentation.
 
+
+Custom Defaults
+---------------
+
+Custom defaults allow us to specify :class:`.Context`-based defaults. Custom defaults are always classes which inherit from
+:class:`.CustomDefault`.
+
+The library provides some simple default implementations in the :module:`default` - :class:`default.Author`, :class:`default.Channel`,
```

Comment:
```suggestion
The library provides some simple default implementations in ``ext.commands.default`` - :class:`.default.Author`, :class:`.default.Channel`,
```

`:module:` is not a valid tag... Is this necessary? (won't make a difference if `:mod:` is used).